SourceX/DiabloUI/diabloui.cpp

```diff
@@ -441,6 +443,8 @@ void UiInitialize()
 			ErrSdl();
 		}
 	}
+
+	prevSgbControllerActive = !sgbControllerActive;

```

Comment:
This assignment seems misleading to me. If the variable is intented to store the "previous value of another", why are we assigning it it's negation instead?

I was expecting to find this somewhere instead:
```c++
prevSgbControllerActive = sgbControllerActive;
...
Change(sgbControllerActive);
...
```